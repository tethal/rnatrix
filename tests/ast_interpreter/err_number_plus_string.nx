fun main() {
    print(42 + " is the answer");
    return 0;
}