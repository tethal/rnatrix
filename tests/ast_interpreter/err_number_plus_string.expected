<string>:2:14: error: operator + cannot be applied to Int and String
    print(42 + " is the answer");
             ^
